automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/previewmodal/PreviewSpec.scala

```diff
@@ -79,7 +79,10 @@ class PreviewSpec extends FreeSpec with WebBrowserSpec with WorkspaceFixtures wi
           //file sometimes changes but is always a JSON array, so easy test...
           filePreview should startWith("[")
           filePreview should endWith("]")
-          previewModal.getPreviewMessage shouldBe "Previews may not be supported for some filetypes."}
+          previewModal.getPreviewMessage shouldBe "Previews may not be supported for some filetypes."
+
+          previewModal.xOut()
```

Comment:
close Preview modal when done verification to avoid annoying `Sign Out` stack trace

Reply:
As I've said before, I think we should stop trying to sign out except when the test actually needs to sign in as a different user.

Reply:
If `page.signOut` is out of `withSignIn`, There is little more work for someone to check all tests, determine whether add `signOut` or not. For example:
```
         withSignIn(owner) { _ =>
            val workspaceDataTab = new WorkspaceDataPage(billingProject, workspaceName).open
            workspaceDataTab.dataTable.hideColumn("test1")
          }
          withSignIn(reader) { _ =>
            val workspaceDataTab = new WorkspaceDataPage(billingProject, workspaceName).open
            workspaceDataTab.dataTable.hideColumn("test2")
          }
          withSignIn(owner) { _ =>
            api.workspaces.waitForBucketReadAccess(billingProject, workspaceName)
```